server.js

```diff
@@ -20,7 +35,27 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello Claudia')
+})
+
+// endpoint returning 20 thoughts in descending order from created date.
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec();
+  res.json(thoughts);
+});
+
+app.post('/thoughts', async (req, res) => {
+  //Retrive the information sent by the client to our API endpoint 
+  const {text, complete} = req.body;
```

Comment:
Ok so I see this endpoint is still under construction 🚧
I understand this "text" is the text we will get from the client (what the use has written in the text field), but not sure what "complete" is.
Hard to tell since the endpoint is not finished, so maybe it becomes more understandable when it's done 🙃